### DISCUSSION TITLE
Documentation for constructor options, e.g. Canvas or Text

### DISCUSSION THREAD

**Discussion by Jan-27** [fabricjs/fabric.js#9124] [Q&A]
Hi there,

is there a documentation for the options Objects passed to the Canvas or Text class? The official docs just say:

options | Object | <optional> | Options object

E.g. setting the initial dimensions in the Canvas constructor.

Cheers,

Jan

EDIT: Or is it just **all** properties?

**Comment by ShaMan123** [comment]
the website is being rewritten
for now best to use TS or VSCODE auto complete available in v6
